src/buildercore/terraform.py

```diff
@@ -445,6 +477,12 @@ def add_table(table_id, table_options):
     if not tf_file['resource']['google_bigquery_table']:
         del tf_file['resource']['google_bigquery_table']
 
+    if not tf_file['data'][DATA_TYPE_HTTP]:
```

Comment:
these are pre-existing bugs, related to services that do not use `data.http` and get an empty block being generated. I have a refactoring to avoid all the setting-deleting of keys, but the PR is getting too big so I'll put it in place later.